daemon/src/model/cfd.rs

```diff
@@ -950,22 +950,20 @@ impl Cfd {
 
     /// Given an attestation, find and decrypt the relevant CET.
     pub fn decrypt_cet(self, attestation: &oracle::Attestation) -> Result<Option<Event>> {
-        anyhow::ensure!(!self.is_final());
+        if self.is_final() {
+            return Ok(None);
+        }
 
         let dlc = match self.dlc.as_ref() {
             Some(dlc) => dlc,
-            None => {
-                tracing::warn!(order_id = %self.id(), "Handling attestation without a DLC is a no-op");
-                return Ok(None);
-            }
+            None => return Ok(None),
         };
 
-        let cet = match dlc.signed_cet(attestation)? {
+        let cet = dlc.signed_cet(attestation)?;
+
+        let cet = match cet {
             Ok(cet) => cet,
-            Err(e @ IrrelevantAttestation { .. }) => {
-                tracing::debug!("{}", e);
-                return Ok(None);
-            }
+            Err(e @ IrrelevantAttestation { .. }) => bail!(e),
```

Comment:
I think this should remain an `Ok(None)`. This will trigger VERY often because we feed every attestation to every CFD.